src/users/user.php

```diff
@@ -0,0 +1,371 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_api
+ *
+ * @copyright   Copyright (C) 2009-2017 Techjoomla, Techjoomla Pvt. Ltd. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access.
+defined('_JEXEC') or die();
+
+/**
+ * User Api.
+ * Creates a new user, updates an existing user and gets data of an user
+ *
+ * @package  Joomla.Plugin
+ *
+ * @since    2.0
+ */
+class UsersApiResourceUser extends ApiResource
+{
+	/**
+	 * Function to create and edit user record.
+	 *
+	 * @return object|void User details on success. raise error on failure.
+	 *
+	 * @since   2.0
+	 */
+	public function post()
+	{
+		$app						= JFactory::getApplication();
```

Comment:
can you equi-align these lines, its hard to read?

Reply:
Done